src/assets/styles/styles/custom/_autocomplete.scss

```diff
@@ -1,39 +1,67 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public 
  * License, v. 2.0. If a copy of the MPL was not distributed with this 
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+.more-left-margin {
+  margin-left: -6px !important;
+}
+
```

Comment:
Chodzi o to, że komórki zawierające początek kwarantanny / urlopu / L4 mają o 4px później element, do którego przyczepiony jest dropdown [dla pozostałych komórek jest `margin-left: -2px`].
![por](https://user-images.githubusercontent.com/48785655/106009678-e197c100-60b8-11eb-83c1-86e86ea486f4.png)
Nie wiem, czy to celowo, czy niedopatrzenie, ale dostosowałam się tak, żeby dropdown wyświetlał się zawsze równo z początkiem uwydatnionej komórki.
![1 base cell component moremarginclass](https://user-images.githubusercontent.com/48785655/106009364-8796fb80-60b8-11eb-9c25-efc5938db6cc.png)
![2 base cell przekazanie w inpucie](https://user-images.githubusercontent.com/48785655/106009375-89f95580-60b8-11eb-976e-89faf241dc8d.png)
![3 autocomplete](https://user-images.githubusercontent.com/48785655/106009386-8e257300-60b8-11eb-850a-1a1d5be75a03.png)
![4 autocomplete](https://user-images.githubusercontent.com/48785655/106009394-91b8fa00-60b8-11eb-813c-87744aa7d634.png)

Reply:
Cóż za dbałość o szczegóły OO To jest niedopatrzenie, więc moim zdaniem powinniśmy raczej to poprawić, a nie dostosowywać się do tego.